tests/test_bot.py

```diff
@@ -2455,3 +2463,25 @@ def test_defaults_warning(self, bot, recwarn, cls, warn):
             assert 'Passing Defaults to telegram.Bot is deprecated.' in str(recwarn[-1].message)
         else:
             assert len(recwarn) == 0
+
+    def test_camel_case_redefinition_extbot(self):
+        invalid_camel_case_functions = []
+        for function_name, function in ExtBot.__dict__.items():
+            camel_case_function = getattr(ExtBot, to_camel_case(function_name), False)
+            if callable(function) and camel_case_function and camel_case_function != function:
```

Comment:
```suggestion
            if callable(function) and camel_case_function and camel_case_function ist not function:
```
just a minor nit pick b/c I have bad experience with relying on `__eq__` to do what you expect it to do :D